qemu/tests/cfg/cpu_device_hotplug_maximum.cfg

```diff
@@ -25,13 +26,11 @@
             vcpu_cores = 0
             vcpu_threads = 1
         - max_thread:
+            only ppc64 ppc64le
```

Comment:
@huangyum Do you also want to skip this one on the intel platform?

Reply:
Yes, threads>2 doesn't match current x86 platform.